### PR TITLE
0.3.3

### LINKED ISSUE #109: `eggs publish` Fails with Deno v1.6.0
## 🐛 Bug Report

When running `eggs publish` on deno v1.6.0, `eggs publish fails with the following error: 

image

## To Reproduce

The steps taken. are here: 

```
runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
      with:
        fetch-depth: 2
        repository: drashland/${{ github.event.client_payload.repo }}
    - uses: denolib/setup-deno@master
    - run: |
        deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.3.2/eggs.ts
        export PATH="/home/runner/.deno/bin:$PATH"
        eggs link ${{ secrets.CI_NESTLAND_API_KEY }}
        eggs publish --no-check-tests --yes
```

## Expected behavior

I understand eggs hasn't been updated, so it's not like i'm expecting this to just work. This issue is more to bring it to your attention. On deno v1.6.0 (eggs v0.3.3?), publishing should work

## Actual Behavior

Publishing failed due to a compiling error

## Environment

- Eggs version: 0.3.2
- Deno version: v1.6.0
- Operating system and version: ubuntu-latest

### PR DESCRIPTION
**Pull request by SteelAlloy**
- Fix: compatibility with Deno 1.6 #109

### CODE REVIEW

**Review by maximousblk — APPROVED** [review]
LGTM

File: src/version.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const version = "0.3.2";
+export const version = "0.3.4";
```
Comment (maximousblk) [inline src/version.ts]:
```suggestion
export const version = "0.3.3";
```

**Review by maximousblk — CHANGES_REQUESTED** [review]
just remembered, last release was `0.3.2` so this will be `0.3.3`

**Review by maximousblk — APPROVED** [review]
LGTM

**Review by t8 — APPROVED** [review]
LGTM

**Pull request merged**